recipes/head_node_slurm_finalize.rb

```diff
@@ -71,15 +71,15 @@
     require 'chef/mixin/shell_out'
     require 'shellwords'
 
-    # $ /opt/slurm/bin/squeue --name='parallelcluster-init-cluster' -O state -h
+    # $ #{node['cfncluster']['slurm']['install_dir']}/bin/squeue --name='parallelcluster-init-cluster' -O state -h
     # CONFIGURING
     # RUNNING
-    are_jobs_running_command = Shellwords.escape("set -o pipefail && /opt/slurm/bin/squeue --name='parallelcluster-init-cluster' -O state -h | { grep -v 'RUNNING' || true; }")
+    are_jobs_running_command = Shellwords.escape("set -o pipefail && #{node['cfncluster']['slurm']['install_dir']}/bin/squeue --name='parallelcluster-init-cluster' -O state -h | { grep -v 'RUNNING' || true; }")
```

Comment:
This line is too long: C: Layout/LineLength: Line is too long. [210/200]
Could you break it into two lines with `\`?